README.md

```diff
@@ -0,0 +1,259 @@
+<a name="readme-top"></a>
```

Comment:
- [ ] Kindly, see the review comments in the backend [review](https://github.com/Movgto/full-stack-app-backend/pull/5). 👍